clang/test/CIR/CodeGen/X86/builtins-x86.c

```diff
@@ -1,13 +1,28 @@
 // Global variables of intergal types

```

Comment:
I think we might need to use `// REQUIRES: x86-registered-target` in the top of this file